source/widget.js

```diff
@@ -62,7 +62,7 @@ Likely.prototype = {
     update: function (options) {
         if (
             options.forceUpdate ||
-            options.url !== this.options.url
+            options.url && options.url !== this.options.url
```

Comment:
Doesn’t it work the same? I.e.

options.url | this.options.url | expression | result
------------ | ------------- | ------------- | -------------
undefined | "ilyabirman.ru" | false && false | false
"ilyabirman.net" | "ilyabirman.ru" | true && false | false
"ilyabirman.ru" | "ilyabirman.ru" | true && false | true

The first two lines always result into `false`, so the additional `options.url` check doesn’t matter. Am I wrong?